plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1249,6 +1249,35 @@ SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/core/apppassword" \
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
 
+#
+# [ Smart Picker ]
+#
+
+# Adding hyperlinks
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/references/resolve$" \
```

Comment:
Maybe missing `+` after `[0-9]`?

Reply:
that was a quick reply, but yeah that's prolly a good idea to add for future proofing